### ISSUE TITLE
Page numbering failing (specific case)

### ISSUE THREAD

**Issue by RossWebb** [sillsdev/ptx2pdf#369]
I am using a change to put in a [new page before each chapter](https://github.com/sillsdev/ptx2pdf/issues/345#issuecomment-764140083). 
That works, however, the 2nd page of each chapter doesn't get a page number. 
It seems that it's not that the program doesn't just not print the number (in the footer) for every 2nd page because if there is only one page of a particular chapter, then the next chapter gets a page number - i.e. it's not an odd/even thing.
Perhaps this is specific to the grep change in the PrintDraftChanges.txt?

**Comment by mhosken** [comment]
I tried it here and apart from some horrendous page breaks (to be resolved) I can't replicate this. Can you send me an archive (See issue #280).

**Comment by RossWebb** [comment]
[BmaSt-DefaultPTXprintArchive.zip](https://github.com/sillsdev/ptx2pdf/files/5902170/BmaSt-DefaultPTXprintArchive.zip)

**Comment by RossWebb** [comment]
Did you see that I put the archive on GitHub?
Ross

[Ross Webb](https://spikenow.com/r/a/?ref=spike-organic-signature&_ts=vtr2c)	[vtr2c]

[Check out Bible Translation in Vanuatu](https://facebook.com/vanuatuBible)

On February 1, 2021 at 3:49 GMT, mhosken <notifications@github.com> wrote:

I tried it here and apart from some horrendous page breaks (to be resolved) I can't replicate this. Can you send me an archive (See issue [#280](https://github.com/sillsdev/ptx2pdf/issues/280)).

**Comment by markpenny** [comment]
Hi @RossWebb - yes, thanks for the Archive. It has helped us replicate the missing page numbers issue, and we hope to have an update available later today. I also noticed that the "(Hide) Verse Numbers" option isn't doing anything useful either - so we'll try to fix that.
The indentation on \\li was probably set too high, so you can change that on the Styles tab to something more sane to get a better look:

![image](https://user-images.githubusercontent.com/12962708/106523230-af1d1680-6545-11eb-8232-4ad25a70a59a.png)

Which results in:

![image](https://user-images.githubusercontent.com/12962708/106523203-a0366400-6545-11eb-8a91-2ef86d55c530.png)

And I hope you don't mind me suggesting... You might want to force the Questions onto their own page so that you always have the story on the left page and the questions on the right page. I wasn't sure if the text Reference \\r needed to be with the story or with the questions, but if you wanted to break before the \\r you can add another line to the PrintDraftChanges.txt file like this:

`  "(\\r .+?\r?\n)" > "\\pb\r\n\1"`

![image](https://user-images.githubusercontent.com/12962708/106524986-2a7fc780-6548-11eb-9bfe-795b5d83035c.png)

**Comment by markpenny** [comment]
> I also noticed that the "(Hide) Verse Numbers" option isn't doing anything useful either - so we'll try to fix that.

Okay - this has been fixed now...

Still working on the missing Page numbers...

**Comment by markpenny** [comment]
@RossWebb Here's a workaround* which will bring the missing page numbers in for you:

On the advanced tab turn on the Apply Customized Settings \(**ptxprint-mods.tex**\), and then click Edit... 

![image](https://user-images.githubusercontent.com/12962708/106550641-6763b280-6578-11eb-9a76-47259dc2b15b.png)

Then paste these two lines into the (empty?) file:
```
\def\RFnoVoddcenter{\pagenumber}
\def\RFnoVevencenter{\pagenumber}
```
It should look like this:

![image](https://user-images.githubusercontent.com/12962708/106550508-24094400-6578-11eb-8162-b7fc0ad30fb2.png)

Hit Print and all should be well... 

* We'll think about the best long term solution to this and incorporate it if we can.

**Comment by RossWebb** [comment]
Ah yes, verse number... I've been meaning to alert you to that one, but keep forgetting!

Little steps forward, eh? Must be so many options/code interacting that it's hard to get everything right the first time, eh?

Ross

[Ross Webb](https://spikenow.com/r/a/?ref=spike-organic-signature&_ts=vv6cf)	[vv6cf]

[Check out Bible Translation in Vanuatu](https://facebook.com/vanuatuBible)

On February 2, 2021 at 1:51 GMT, Mark Penny <notifications@github.com> wrote:

I also noticed that the "(Hide) Verse Numbers" option isn't doing anything useful either - so we'll try to fix that.

Okay - this has been fixed now...

Still working on the missing Page numbers...

**Comment by markpenny** [comment]
Okay, this (pg num fix) will be fixed in 1.5.9.1 (available later today). You'll be able to remove the .tex file workaround.

**Comment by RossWebb** [comment]
Wow, you guys are good and fast! Thanks for the other clues.
Ross

[Ross Webb](https://spikenow.com/r/a/?ref=spike-organic-signature&_ts=vv6wg)	[vv6wg]

[Check out Bible Translation in Vanuatu](https://facebook.com/vanuatuBible)

On February 2, 2021 at 4:15 GMT, Mark Penny <notifications@github.com> wrote:

Okay, this (pg num fix) will be fixed in 1.5.9.1 (available later today). You'll be able to remove the .tex file workaround.

**Comment by markpenny** [comment]
If you want it to split AFTER the reference, then use these 2 lines instead:

```
 "(\\c \d+ ?\r?\n)" > "\\pb\r\n\1"
 "(\\s Questions ?\r?\n)" > "\\pb\r\n\1"
```
But note that some storied then spill over to the 3rd page. I'd recommend decreasing the top margin to ensure that each story fits on the page along with the reference.
